### PR TITLE
Feature/bulk mute

### PR DESCRIPTION
**Pull request by Bogdan-D**
# Submit a pull request

## CLA

- [x] I have signed the [Stream CLA](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) (required).
- [x] The code changes follow best practices
- [x] Code changes are tested (add some information if not applicable)

### CODE REVIEW

File: user.go [hunk 1]
```diff
@@ -71,6 +90,25 @@ func (c *Client) UnmuteUser(targetID string, userID string) error {
 	return c.makeRequest(http.MethodPost, "moderation/unmute", nil, data, nil)
 }
 
+// Removes a mute
+// targetID: the user getting un-muted
+// userID: the user muting the target
+func (c *Client) UnmuteUsers(targetIDs []string, userID string) error {
+	switch {
```
Comment (tbarbugli) [inline user.go]: can't use a switch here (unless you use the fall through but that's ugly)

Comment (tbarbugli) [reply user.go]: (both params needs validation)

File: user.go [hunk 2]
```diff
@@ -71,6 +90,25 @@ func (c *Client) UnmuteUser(targetID string, userID string) error {
...
+	case len(targetIDs) == 0:
+		return errors.New("target IDs is empty")
+	case userID == "":
+		return errors.New("user ID is empty")
+	}
+
+	data := map[string][]string{
+		"target_ids": targetIDs,
+		"user_id":    {userID},
```
Comment (tbarbugli) [inline user.go]: why are you sending `user_id` as []string? either server-side we do something weird or this client is weird

File: user.go [hunk 3]
```diff
@@ -52,13 +52,32 @@ func (c *Client) MuteUser(targetID string, userID string) error {
...
+}
+
 // Removes a mute
 // targetID: the user getting un-muted
 // userID: the user muting the target
 func (c *Client) UnmuteUser(targetID string, userID string) error {
 	switch {
-	case targetID == "":
-		return errors.New("target ID is empty")
+	case len(targetID) == 0:
```
Comment (tbarbugli) [inline user.go]: previous check was better

File: user.go (hunk 2 above)
Comment (Bogdan-D) [reply user.go]: @tbarbugli `url.Values` does have signature `map[string][]string`

File: user.go (hunk 3 above)
Comment (Bogdan-D) [reply user.go]: don't remember why this was changed

File: user.go (hunk 1 above)
Comment (Bogdan-D) [reply user.go]: switch looks cleaner than many if's, but it depends on reviewer

Comment (Bogdan-D) [reply user.go]: but ofc its not problem to just write multiple `if`.

**Pull request merged**